abnf/odata-abnf-testcases.yaml

```diff
@@ -2164,6 +2166,19 @@ TestCases:
     Rule: odataRelativeUri
     Input: ProductsOrderedBy(Customers=@c)?@c=[$root/Customers('ALFKI'),$root/Customers('BLAUS')]
 
+  - Name: 5.1.1.14.5 $root with entity set
+    Rule: commonExpr
+    Input: $root/SalesOrganizations
+
+  - Name: $root with bound function call
+    Rule: odataRelativeUri
+    FailAt: 68
+    Input: Sales?$filter=$root/Sales/Special.MostPopularCategory(Where=ZipCode) eq 'Food'
+
+  - Name: $root with unbound function call
+    Rule: odataRelativeUri
+    Input: Sales?$filter=Special.MostPopularCategory(What=$root/Sales,Where=ZipCode) eq 'Food'
+
```

Comment:
`$root` with unbound function call allows `commonExpr` as parameter value:
https://github.com/oasis-tcs/odata-abnf/blob/5c9b9f15ff22aee469c5a0cf2a01f9f4fec46dfa/abnf/odata-abnf-construction-rules.txt#L587

whereas `$root` with bound function call only allows literals or aliases:
https://github.com/oasis-tcs/odata-abnf/blob/5c9b9f15ff22aee469c5a0cf2a01f9f4fec46dfa/abnf/odata-abnf-construction-rules.txt#L224

Is this different treatment really justified?

Reply:
This needs to be fixed

Reply:
Fixed.